spec/requests/api/v1/user_book_show_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'GET /api/v1/users/:id/books/:id' do
+  it 'returns paginated gutenberg book' do
+    user = create(:user)
+    book_1 = create(:book, guten_id: 11)
+    book_2 = create(:book, guten_id: 345)
+    user_book1 = create(:user_book, user: user, book: book_1, current_page: 10)
+    user_book2 = create(:user_book, user: user, book: book_2, current_page: 23)
+
+    get "/api/v1/users/#{user.id}/books/#{book_1.id}"
+
+    expect(response.status).to eq(200)
+
+    data = JSON.parse(response.body, symbolize_names: true)
+
+    expect(data).to have_key(:data)
+    expect(data[:data]).to have_key(:current_page)
+    expect(data[:data]).to have_key(:book)
+
+    expect(data[:data][:current_page]).to equal(10)
+    expect(data[:data][:book].count).to equal(167)
+
+    first_page_text = "Project Gutenberg’s Alice’s Adventures in Wonderland, by Lewis Carroll\r\n\r\nThis eBook is for the use of anyone anywhere at no cost and with almost no restrictions whatsoever.  You may copy it, give it away or re-use it under the terms of the Project Gutenberg License included with this eBook or online at www.gutenberg.org\r\n\r\n\r\nTitle: Alice’s Adventures in Wonderland\r\n\r\nAuthor: Lewis Carroll\r\n\r\nPosting Date: June 25, 2008 [EBook #11] Release Date: March, 1994 Last Updated: October 6, 2016\r\n\r\nLanguage: English\r\n\r\nCharacter set encoding: UTF-8\r\n\r\n*** START OF THIS PROJECT GUTENBERG EBOOK ALICE’S ADVENTURES IN WONDERLAND ***\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\nALICE’S ADVENTURES IN WONDERLAND\r\n\r\nLewis Carroll\r\n\r\nTHE MILLENNIUM FULCRUM EDITION 3.0\r\n\r\n\r\n\r\n\r\nCHAPTER I. Down the Rabbit-Hole\r\n\r\nAlice was beginning to get very tired of sitting by her sister on the bank, and of having nothing to do: once or twice she had peeped into the book her sister was reading, but it had no pictures or conversations in it,"
+
+    last_page_text = " PG search facility:\r\n\r\n     http://www.gutenberg.org\r\n\r\nThis Web site includes information about Project Gutenberg-tm, including how to make donations to the Project Gutenberg Literary Archive Foundation, how to help produce our new eBooks, and how to subscribe to our email newsletter to hear about new eBooks.\r\n"
+
+    expect(data[:data][:book][0]).to eq(first_page_text)
+    expect(data[:data][:book].last).to eq(last_page_text)
+
+    data[:data][:book].each do |page|
+      expect(page.length).to be < (1025)
+    end
```

Comment:
This seems a little over-tested. I think just asserting that the number of pages is correct along with a visual inspection of the output should be sufficient.

Reply:
I created this to know that the pages are being parsed constantly. So each time the test is ran (each time the parsed service is called) we can expect the same first/last page at least. The character could test I created to double check each page will be no more than 1025 characters. Wanted to make sure we could tell frontend to expect less than 1025 per page.

Reply:
That makes sense. It will just be a bit of a pain to change what to expect from lines 24, 26 if we change the character count per page, like we should in my other comment.